undercover.el

```diff
@@ -386,14 +395,48 @@ Values of that hash are number of covers."
     (shut-up
      (shell-command
       (format "curl -v -include --form json_file=@%s %s" undercover--report-file-path coveralls-url)))
-    (message "Sending: OK")))
+    (message "Sending: ----------> OK")))
 
 (defun undercover--coveralls-report ()
   "Create and submit test coverage report to coveralls.io."
   (undercover--save-coveralls-report (undercover--create-coveralls-report))
   (when undercover--send-report
     (undercover--send-coveralls-report)))
 
+
+;; ------------
+;; Text report
+;; ------------
+
+
+(defun undercover--create-text-report ()
+  "Create test coverage report for text display."
+  (undercover--collect-files-coverage undercover--files)
+  (let ((report (make-hash-table :test 'equal)))
+    (maphash (lambda (key val)
```

Comment:
Please extract both lambdas to separated functions and give arguments meaningful name. To be honest it's hard to understand what `k, v, nb` means.